cmd/validate/image.go

```diff
@@ -40,20 +41,24 @@ type imageValidationFunc func(context.Context, string, policy.Policy, bool) (*ou
 
 func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 	var data = struct {
-		effectiveTime       string
-		filePath            string
-		imageRef            string
-		info                bool
-		input               string
-		output              []string
-		outputFile          string
-		policy              policy.Policy
-		policyConfiguration string
-		publicKey           string
-		rekorURL            string
-		snapshot            string
-		spec                *app.SnapshotSpec
-		strict              bool
+		certificateIdentity         string
+		certificateIdentityRegExp   string
+		certificateOIDCIssuer       string
+		certificateOIDCIssuerRegExp string
```

Comment:
Should we consider providing this (also?) as inline JSON, worse for UX but better for flexibility if `cosign.Identity` changes?

Reply:
https://issues.redhat.com/browse/HACBS-2073 should take care of that.